imagersite/templates/imager_profile/equipment_template.html

```diff
@@ -0,0 +1,9 @@
+{% extends "imagersite/base.html" %}
+{% block body %}
+
+    <h3>Type: {{ potographer.equipment.type_}}</h3>
```

Comment:
you should really do some spell checking. The template that calls in this snippet uses the name "photographer", not "potographer".